tools/templates/lint-and-validate-values.yaml

```
@@ -144,6 +144,9 @@ proxy:
       interNamespaceAccessLabels: accept
       allowedIngressPorts: [http, https]
   traefik:
+    extraPorts:
+    - name: ssh
+      containerPort: 22
```

Lets examplify with a port number over 1000 to be compliant with strict pod security policies.

```suggestion
      containerPort: 8022
```